lib/kafka.ts

```diff
@@ -58,3 +58,75 @@ export const waitForKafkaReady = async function (page: Page, name: string) {
     timeout: config.kafkaInstanceCreationTimeout
   });
 };
+
+export const manageAccess = async function (page: Page, id: string, desc: string) {
+  await page.getByTestId('actionManagePermissions').click();
+
+  await page.getByRole('button', { name: 'Options menu' }).click();
+
+  await page.getByRole('option', { name: id + ' ' + desc }).click();
+
+  await page.getByRole('button', { name: 'Next' }).click();
+
+  await page.getByTestId('permissions-dropdown-toggle').click();
+
+  await page
+    .getByRole('menuitem', {
+      name: 'Produce to a topic Provides access to produce to one or more topics depending on topic selection criteria'
+    })
+    .click();
+
+  await page.getByPlaceholder('Enter prefix').click();
+
+  await page.getByPlaceholder('Enter prefix').fill('test');
+
+  await page.getByTestId('permissions-dropdown-toggle').click();
+
+  await page
+    .getByRole('menuitem', {
+      name: 'Consume from a topic Provides access to consume from one or more topics depending on topic and consumer group selection criteria'
+    })
+    .click();
+
+  await page
+    .getByRole('row', {
+      name: 'T Topic Options menu permission.manage_permissions_dialog.assign_permissions.resource_name_aria Options menu Label group category'
+    })
+    .getByPlaceholder('Enter prefix')
+    .click();
+
+  await page
+    .getByRole('row', {
+      name: 'T Topic Options menu permission.manage_permissions_dialog.assign_permissions.resource_name_aria Options menu Label group category'
+    })
+    .getByPlaceholder('Enter prefix')
+    .fill('test');
+
+  await page
+    .getByRole('gridcell', {
+      name: 'permission.manage_permissions_dialog.assign_permissions.resource_name_aria Options menu'
+    })
+    .getByPlaceholder('Enter prefix')
+    .click();
+
+  await page
+    .getByRole('gridcell', {
+      name: 'permission.manage_permissions_dialog.assign_permissions.resource_name_aria Options menu'
+    })
+    .getByPlaceholder('Enter prefix')
+    .fill('test');
+
+  await page.getByRole('button', { name: 'Save' }).click();
+};
+
+export const navigateToAccess = async function (page: Page, kafkaName: string) {
+  await page.locator('button:has-text("Streams for Apache Kafka")').click();
+  await page.locator('a:has-text("Kafka Instances")').click();
+  await expect(page.getByText(kafkaName)).toHaveCount(1);
+  await page.getByText(kafkaName).click();
+  await page.getByTestId('pageKafka-tabPermissions').click();
+};
+
+export const navigateToConsumerGroups = async function (page: Page) {
+  await page.getByTestId('pageKafka-tabConsumers').click();
```

Comment:
Same here.

Reply:
Replaced for `await page.getByText('Consumer groups').click();`

Reply:
Can I resolve this conversation? @mmusil